_themes/slides/static/js/jquery.min.js

```diff
@@ -1,4 +0,0 @@
-/*! jQuery v1.11.1 | (c) 2005, 2014 jQuery Foundation, Inc. | jquery.org/license */
```

Comment:
My only question/concern is about removing this file. I am trying to get this running locally but my machine can't seem to get it going. So does everything seem to function correctly even after removing this file? If so then I'm all 👍